lib/redux/constants/errors.js

```diff
@@ -30,5 +30,6 @@ module.exports = {
   E_READ_FILE: 'Error reading file ',
   E_SERIAL_CONNECTION: 'Hmm, we couldn\'t detect your device',
   E_SERVER_ERR: 'Sorry, the Tidepool servers appear to be down',
-  E_UPLOAD_IN_PROGRESS: 'Sorry, an upload is already in progress'
-};
\ No newline at end of file
+  E_UPLOAD_IN_PROGRESS: 'Sorry, an upload is already in progress',
+  E_DEXCOM_UNSUPPORTED: 'Sorry, we don\'t support this Dexcom device yet'
```

Comment:
This is the first time that I'm adding an error message that only appears if you click on Error Details to this list, instead of just in-line. `Something went wrong during device upload` is the higher-level "checked off" error message that gets displayed. I don't think we should get sign off for all error detail messages. I'm not even entirely sure I should be putting it in this file - maybe we need a separate file for error detail messages?